nss-day-cohort-39/hello-world-wanderlust
# Nav bar header

CHANGES
-Added a header to the navigation bar that reads "Wanderlust" and links to index.html

TO TEST
-Ensure "Wanderlust" header appears on every page
-Ensure link to index.html works
-Resize window and ensure nothing crazy happens
